testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/cdi/interceptors/resource/NameBoundCDIProxiesApplication.java

```diff
@@ -10,15 +9,8 @@
 @NameBoundProxiesAnnotation
 public class NameBoundCDIProxiesApplication extends Application {
 
+   @Override
    public Set<Class<?>> getClasses() {
-      Set<Class<?>> set = new HashSet<Class<?>>();
-      set.add(NameBoundCDIProxiesResource.class);
-      return set;
-   }
-
-   public Set<Object> getSingletons() {
-      Set<Object> set = new HashSet<Object>();
-      set.add(new NameBoundCDIProxiesInterceptor());
-      return set;
+      return Set.of(NameBoundCDIProxiesResource.class, NameBoundCDIProxiesInterceptor.class);
```

Comment:
There is another way to fix this test too by annotating `NameBoundCDIProxiesInterceptor` with `@Vetoed`. Since these had `CDI` in the name though, I opted to treat it as a CDI bean. The reason it requires this change is `getSingletons()` returns what are supposed to be singletons which is not a CDI bean.